src/wasm32_bindgen.rs

```diff
@@ -88,6 +94,11 @@ extern "C" {
     fn get_self() -> Result<Self_, JsValue>;
 
     type Self_;
+
+    // Internet Explorer 11
+    #[wasm_bindgen(method, getter, js_name="msCrypto", structural)]
+    fn ms_crypto(me: &Self_) -> JsValue;
+
```

Comment:
Formatting nit here:

```rust
    type Self_;
    #[wasm_bindgen(method, getter, structural)]
    fn crypto(me: &Self_) -> JsValue;
    #[wasm_bindgen(method, getter, js_name = "msCrypto", structural)]
    fn ms_crypto(me: &Self_) -> JsValue;
```

We should explain the use of `msCrypto` above (when we use it)